nbcparser/pkg/parser/templates/containerd.toml.gtpl

```diff
@@ -45,7 +45,7 @@ root = "{{.KubeletConfig.GetContainerDataDir}}"{{- end}}
     {{- end}}
     {{- if getIsKrustlet .GetWorkloadRuntime }}
     [plugins."io.containerd.grpc.v1.cri".containerd.runtimes.spin]
-      runtime_type = "io.containerd.spin-v0-3-0.v1"
```

Comment:
I'm assuming this is updating the "default"/"latest" spin version? will updating this still allow v0-3-0 to be used as specified by RP?

Reply:
Yes. If the `spin` runtime is used, it will use the latest version. If the `spin-v0-3-0` is used, then it would use v0-3-o.